examples/run_gesture_detection.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+"""
+Real time detection of 6 hand gestures.
+
+Usage:
+  run_gesture_detection.py [--camera_id=CAMERA_ID]
+                           [--path_in=FILENAME]
+                           [--path_out=FILENAME]
+                           [--title=TITLE]
+                           [--model_name=NAME]
+                           [--model_version=VERSION]
+                           [--use_gpu]
+  run_gesture_detection.py (-h | --help)
+
+Options:
+  --path_in=FILENAME         Video file to stream from
+  --path_out=FILENAME        Video file to stream to
+  --title=TITLE              This adds a title to the window display
+  --model_name=NAME          Name of the model to be used.
+  --model_version=VERSION    Version of the model to be used.
+  --use_gpu                  Whether to run inference on the GPU or not.
+"""
+from docopt import docopt
+
+import sense.display
+from sense.controller import Controller
+from sense.downstream_tasks.gesture_detection import LAB2INT
+from sense.downstream_tasks.gesture_detection import INT2LAB
+from sense.downstream_tasks.gesture_detection import ENABLED_LABELS
+from sense.downstream_tasks.gesture_detection import LAB_THRESHOLDS
+from sense.downstream_tasks.nn_utils import LogisticRegression
+from sense.downstream_tasks.nn_utils import Pipe
+from sense.downstream_tasks.postprocess import EventCounter
+from sense.downstream_tasks.postprocess import PostprocessClassificationOutput
+from sense.loading import get_relevant_weights
+from sense.loading import build_backbone_network
+from sense.loading import ModelConfig
+
+
+SUPPORTED_MODEL_CONFIGURATIONS = [
+    ModelConfig('StridedInflatedEfficientNet', 'pro', ['gesture_detection']),
+    ModelConfig('StridedInflatedEfficientNet', 'lite', ['gesture_detection']),
+]
+
+if __name__ == "__main__":
+    # Parse arguments
+    args = docopt(__doc__)
+    camera_id = int(args['--camera_id'] or 0)
+    path_in = args['--path_in'] or None
+    path_out = args['--path_out'] or None
+    title = args['--title'] or None
+    model_name = args['--model_name'] or None
+    model_version = args['--model_version'] or None
+    use_gpu = args['--use_gpu']
+
+    # Load weights
+    selected_config, weights = get_relevant_weights(
+        SUPPORTED_MODEL_CONFIGURATIONS,
+        model_name,
+        model_version
+    )
+
+    # Load backbone network
+    backbone_network = build_backbone_network(selected_config, weights['backbone'],
+                                              weights_finetuned=weights['gesture_detection'])
```

Comment:
I want something that does not break on Travis so I ended up using `update_backbone_weights` within `build_backbone_network` and switching to `build_backbone_network` in `run_custom_classifier` and `train_classifier`
Let me know what you think

Reply:
Looks good :+1: I was also able to remove the `strict=False` below now, because `update_backbone_weights` pops the fine-tuned layers from the weights dictionary.